ci/tests/general/general_disk.py

```diff
@@ -196,6 +199,55 @@ def partition_disk(disk):
         return disk.partitions[0]
 
     @staticmethod
+    def unpartition_disk(disk=None, partitions=None, wait=True):
+        """
+        Return disk to RAW state
+        :param disk: Disk DAL object
+        :param partitions: Partitions DAL object list
+        :return: None
+        """
+        if disk is None and partitions is None:
+            return
```

Comment:
I'd raise in this case, because you're not gonna make this call without providing any of the 2 --> useless